src/main/java/org/openstreetmap/atlas/tags/BuildingLevelsTag.java

```diff
@@ -14,4 +17,17 @@
 {
     @TagKey
     String KEY = "building:levels";
+
+    DoubleValidator validator = new DoubleValidator();
```

Comment:
Per the wiki, "ground_level" is an accepted value. It might be worthwhile having a special case for it, there are over 1,000 such values. Source: https://taginfo.openstreetmap.org/tags/building%3Alevels=Ground%20Level

Reply:
Chatted offline, I'm incorrect here, misread the wiki. Please ignore above comment.